### DISCUSSION TITLE
Undefined variable $parentLocation in YellowMedia::getParentLocation() and YellowMedia::getParentTopLocation()

### DISCUSSION THREAD

**Discussion by jeromehode** [datenstrom/community#992] [Report a bug] [answered]
Hi,
IMHO, a declaration/initialization of $parentLocation is missing in [YellowMeda.getParentLocation()](https://github.com/datenstrom/yellow/blob/aa80148bf382db66f8b7812af64532876268f421/system/workers/core.php#L604) and 
[YellowMeda.getParentTopLocation()](https://github.com/datenstrom/yellow/blob/aa80148bf382db66f8b7812af64532876268f421/system/workers/core.php#L614), causing is_string_empty() to warn for some codepath in debug mode (so not that critical...)

**Comment by markseuffert** [comment]
Yes, looks like the initialisation is missing. Can you tell me which PHP version you are using?

**Comment by jeromehode** [comment]
PHP 8.1.29 (cli) (built: Jul 19 2024 10:32:22) (NTS)

**Comment by markseuffert** [comment, marked as answer]
Thank you for reporting a bug in Core 0.9.11. Should be fixed in [latest core](https://github.com/annaesvensson/yellow-core), please try it out.
